openerp/addons/base/tests/base_test.yml

```diff
@@ -92,56 +92,6 @@
         name: Child 2-1
         parent_id: test_categ_2
 -
-    2. Duplicate the parent category and verify that the children have been duplicated too and are below the new parent
```

Comment:
Here we making category as a unique and this test case is for duplicate the parent category and verify that the children have been duplicated too and are below the new parent.
So now we are not allowed to create duplicate category so i remove this tests as no more needed.